src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1022,19 +1083,19 @@ ExpressionPtr Visitor::visitTypesafe(Parser::PrimaryExpressionContext* ctx) {
     auto literalAny = visitNumericLiteral(ctx->numericLiteral());
     try {
       auto intLiteral = literalAny.as<unsigned long long>();
-      return ExpressionPtr{std::make_unique<sparqlExpression::IntExpression>(
+      return {std::make_unique<sparqlExpression::IntExpression>(
```

Comment:
Also redundant braces, but I think this horrible function (written by me:)) is being rewritten soon anyway if I understood you correctly.

Reply:
Yes that's the goal.